kolibri/core/auth/backends.py

```diff
@@ -6,7 +6,7 @@
 from kolibri.core.auth.models import FacilityUser
 
 
-FACILITY_CREDENTIAL_KEY = "facility"
+FACILITY_CREDENTIAL_KEY = "facility_id"
```

Comment:
I remember when @bjester first implemented this, he had it as `facility_id` and then changed it back. At the moment this will break login, as it is still passing `facility`.

If possible, I'd prefer we not include this change, and just pass `facility` for the facility id in the authentication parameters.

Reply:
in such case authentication must be changed because using `facility` just does not work. That's why I changed it

Reply:
This part https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/auth/management/utils.py#L238-L249 does not work if `facility` is used, but it does using `facility_id`

Reply:
Ah yes, I think this code was the reason that @bjester originally had it set to `facility_id` so I'd like to understand why he switched it back!

Reply:
@jredrejo There's an [integration test](https://github.com/learningequality/kolibri/blob/release-v0.15.x/kolibri/core/auth/test/test_morango_integration.py#L76) for this which verifies that code you linked does actually work as it was prior to your changes. I'm not sure what issue you're running into. If you look [here](https://github.com/learningequality/kolibri/blame/40dcc2408b86980e3d17720da476725c28792fd2/kolibri/core/auth/backends.py#L14), you can see that `FacilityUserBackend` was originally accepting a `facility` keyword argument. So changing `facility` to `facility_id` could be breaking more functionality, since support for `facility` was existing prior to any 0.15 change

Reply:
commit removed!